setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="torrent-tracker-scraper",
-    version="3.1.2",
```

Comment:
@49e94b8f256530dc0d41f740dfe8a4c1 still think that you don't like 3 ;)

Reply:
I intended for the PRs to be merged in the versioning order because of the Jenkins. That's why the titles have, or had a version prefix.

Reply:
Oh... You're right. I missed that ☺️

Reply:
The actual problem is I did not foresee.multilple PRs affecting infrastructure being open at the same time. I therefore think we should move the versioning to develop and leave the releases on master because we have established that manually bumping via PRs is confusing :)

Reply:
I thought about the same thing and wanted to talk to you tomorrow 😅 yeah! Let's do that 👍